spacepy/coordinates.py

```diff
@@ -2,97 +2,12 @@
-    of ``ECIMOD``, these can be used with the existing IRBEM backend, and
-    will be converted to their closest equivalents.
-
-    .. versionchanged:: 0.4.0
-
-    The default backend for coordinate transformations was changed from IRBEM
-    to the CTrans-based SpacePy backend.
-
-Notes on differences between representations
---------------------------------------------
-IRBEM's coordinate transformations are low-accuracy and were written for
-a library with a driving philosophy of speed and robustness as priorities.
-The coordinate transformations are therefore approximate. Further, most of
-the geophysical systems (e.g., GSE, SM) are derived from an inertial
-system. It is standard practice to use ECIMOD as this system. However,
-IRBEM does not currently make ECIMOD available as one of its inertial
-systems. IRBEM's default inertial system (called GEI) is consistent with
-an approximation of ECITOD. Hence there will be small differences between
-IRBEM's transformations and those using SpacePy's CTrans backend.
-Further sources of difference include: IRBEM uses a low-order approximation
-to the sidereal time and other parameters; the calculation of the Earth-Sun
-vector differs between the representations; the definitions of an Earth
-radius differ (SpacePy = 6378.137km; IRBEM = 6371.2 km). SpacePy's in-built
-representation is higher accuracy and is comprehensively tested, including
-tests for consistency with other high accuracy packages such as LANLGeoMag
-and AstroPy. However, for use cases where the required precision is of order
-1 percent the output can be considered equivalent.
-
-Setting options for coordinate transformation
----------------------------------------------
-The backend for coordinate transformations can be provided at
-instantiation of a :class:`Coords` object using a keyword
-argument. However, for convenience and flexibility the options can be
-set at the module level. Configurable options include the backend used
-(:mod:`~spacepy.irbempy` or SpacePy's :mod:`~spacepy.ctrans`) and the
-reference ellipsoid (only configurable for the SpacePy backend). A
-warning will be raised if the backend is not set (either through the
-defaults or the keyword argument). The final configurable option
-(``itol``) is the maximum separation, in seconds, for which the
-coordinate transformations will not be recalculated. To force all
-transformations to use an exact transform for the time, set ``itol``
-to zero. Values between 10s and 60s are recommended for speed while
-also preserving accuracy, though different applications will require
-different accuracies.  For example, assuming this module has been
-imported as ``spc``, to set the SpacePy backend as the default and set
-``itol`` to 5 seconds:
-
-    >>> spc.DEFAULTS.set_values(use_irbem=False, itol=5)
-
 Authors: Steven Morley and Josef Koller
 Institution: Los ALamos National Laboratory
 Contact: smorley@lanl.gov
 
+For additional documentation :doc:`../coordinates`
```

Comment:
Does this not work as ``:doc:`coordinates`?`` If not, probably should be `/coordinates` to be explicitly relative to the root. Or maybe there should be an explicit reference reference anchor added there and this can be `:ref:` If this is rendering with the title then it's probably okay to stay as `:doc:` with the absolute ref.

Reply:
Yeah, I tried `:ref:` at first and it didn't seem to work, `:doc:` worked and renders with the title correctly

Reply:
ref doesn't work unless you [give it a reference label](https://www.sphinx-doc.org/en/master/usage/referencing.html#role-ref).

Reply:
I just left it as :doc:`/coordinates` which worked fine, and I think simpler than adding reference labels, although possibly not as powerful.